src/Qwiq.Mapper/packages.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="FastMember" version="1.1.0" targetFramework="net46" />
-  <package id="GitVersionTask" version="4.0.0-beta0012" targetFramework="net46" developmentDependency="true" />
+  <package id="GitVersionTask" version="4.0.0" targetFramework="net46" developmentDependency="true" />
   <package id="Microsoft.Net.Compilers" version="2.8.2" targetFramework="net46" developmentDependency="true" />
+  <package id="UtilPack.NuGet.MSBuild" version="2.8.0" targetFramework="net46" developmentDependency="true" />
```

Comment:
Ah! Yes, ok, that's what it was. Ok, I wonder if that will be cause issues where projects that use project references aren't going to have the .props files which are brought in by that package. If it seems like it isn't needed for the other projects then everything looks good otherwise.